float64[6] B_F_ext
